src/components/Notifications/NotificationFeed/NotificationFeed.js

```diff
@@ -527,12 +529,11 @@ class NotificationFeed extends React.Component {
   }
 
   renderDeleteAllButton = () => {
-    if (!this.props.enableFetchAllNotificationFeedAcrossProjects) {
-      return
+    let title = `Delete all notifications for ${this.props.selectedProjectName}?`
```

Comment:
Maybe doing this where you set the projectName make more sense. For example, 
`const projectName = this.props.selectedProjectName === 'All' ? 'all data sources' : this.props.selectedProjectName

const title = `Delete all notifications for ${projectName}?`
`
What do you think?

Reply:
good idea!